crates/server/Cargo.toml

```diff
@@ -15,8 +15,8 @@ publish = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-arrow = "10.0.0"
-arrow-flight = "10.0.0"
+arrow = "20.0.0"
+arrow-flight = "20.0.0"
```

Comment:
like, skipping 10 major release, because why not 😅

Reply:
That's the :arrow_double_up: way